Libplanet.Headless/Hosting/LibplanetNodeService.cs

```diff
@@ -178,6 +172,15 @@ public class LibplanetNodeService<T> : IHostedService, IDisposable
 
         public virtual async Task StartAsync(CancellationToken cancellationToken)
         {
+            Log.Debug("Trying to delete {count} obsoleted chains...", _obsoletedChainIds.Count());
+            _ = Task.Run(() =>
+            {
+                foreach (Guid chainId in _obsoletedChainIds)
+                {
+                    Store.DeleteChainId(chainId);
+                    Log.Debug("Obsoleted chain[{chainId}] has been deleted.", chainId);
+                }
+            });
```

Comment:
Is there a reason to assign the value to `_`?

Reply:
`_ = Task.Run()` is [a standalone discard](https://docs.microsoft.com/en-us/dotnet/csharp/discards#a-standalone-discard) for fire-forget pattern. we don't want to wait for this task and we don't care about it.